src/core/Messages.ts

```diff
@@ -204,12 +209,15 @@ export interface TradeFinalizedMessage extends StreamMessage {
     reason: string;
 }
 
+/**
+ * Emitted when my order is placed. (An authenticated feed is
+ * required).
+ */
 export interface MyOrderPlacedMessage extends StreamMessage {
     productId: string;
     orderId: string;
     side: string;
     price: string;
-    orderType: string;
```

Comment:
Why is this removed?

Reply:
@fb55 Looking at the actual feed, when the type is 'open' there is no orderType. The commit message describes the reason for this.